### PR TITLE
feat: ⚡️ add pride filter

### LINKED ISSUE #103: [feat] gallery page/link for TezosPride event
**Is your feature request related to a problem? Please describe.**
Could we please have a gallery/event page/link for Tezos Pride event, similar to the Ukraine Fundraiser link (which was PR45 https://github.com/teia-community/teia-ui/pull/45 for reference)

This event may need to use the event tag instead of contract id like UA fundraiser query did 

Mal has mentioned NFTBiker had an option to add list of work to a spreadsheet and have those included in the event page, eg for those who forget to use the event tag

**Describe the solution you'd like**
A link for the Tezos Pride event on Teia homepage that links to a page showing the artwork tagged with event tag, similar to the Ukraine link
image

**Describe alternatives you've considered**
The alternative is to use the gallery feature and manually update JSON file, though this is not a dynamic list of work, it would need manual operations. This would cater for any not using the event tag. for this case, there'd be a new gallery in the Gallery page rather than a link on homepage (so different to UA fundraiser event)

Please advise if this is preferred option over the dynamic link above

**Additional context**
please contact MaliciousSheep or AliaK in discord

initial discussion for this request: https://discord.com/channels/908672304236625970/951351576839221248/982983061610319944

discord link for TezosPride event planning: https://discord.com/channels/908672304236625970/981374833642328084

### PR DESCRIPTION
**Pull request by melMass**

### CODE REVIEW

File: src/pages/search/index.js [hunk 1]
```diff
@@ -179,43 +179,6 @@ async function fetchMusic(offset) {
   }
 }
 
-async function fetchDay(day, offset) {
```
Comment (xat) [inline src/pages/search/index.js:182]: 👍 for removing code :)

File: src/pages/search/index.js [hunk 2]
```diff
@@ -287,7 +250,7 @@ async function fetchSubjkts(subjkt) {
 async function fetchTag(tag, offset) {
   const { errors, data } = await fetchGraphQL(
     `query ObjktsByTag {
-  token(where: {supply : { _neq : 0 }, token_tags: {tag: {tag: {_eq: ${tag}}}}, id: {_lt: ${offset}}}, limit : 15, order_by: {id: desc}) {
+  token(where: {token_tags: {tag: {tag: {_eq: "${tag}"}}}, supply: {_neq: "0"}}, offset: ${offset}, limit: 15, order_by: {id: desc}) {
```
Comment (xat) [inline src/pages/search/index.js:253]: no need to change, but i think the code would be more robust if we would be using graphql variables in cases like these.

File: src/pages/search/index.js [hunk 3]
```diff
@@ -400,9 +331,13 @@ export class Search extends Component {
         'creator_id'
       ),
     })
-    //this.latest(999999)
   }
-
+  componentDidMount = async () => {
```
Comment (xat) [inline src/pages/search/index.js]: why is it async?

**Review by xat — APPROVED** [review]

File: src/pages/search/index.js (hunk 2 above)
Comment (melMass) [reply src/pages/search/index.js:253]: Definitely!

File: src/pages/search/index.js (hunk 3 above)
Comment (melMass) [reply src/pages/search/index.js]: My bad, I though it was a requirement of CdM

**Pull request merged**